src/components/company/chart.js

```diff
@@ -112,11 +114,11 @@ const tickStyle = {
 
 const DisplayChart = ({ priceFilter, history }) => {
   return (
-    <ResponsiveContainer width="100%" height={500}>
+    <ResponsiveContainer width="99%" height="100%">
```

Comment:
Would it be possible to move the width and height values to the CSS file for example if one wanted to change the width and height of the responsivecontainer, one had to look at the CSS file rather than in the js file. Separation of Concern, I believe